strace_process_tree.py

```diff
@@ -297,7 +327,7 @@ def _format(self, theme, processes, indent='', level=0):
                 self.exit_time.get(process),
             )
             title = '{pid} {name} {time_range}'.format(
-                pid=theme.pid(process.pid or '<unknown>'),
+                pid=theme.pid(str(process.pid or '<unknown>')),
```

Comment:
Hm.  I'd rather the theme methods accepted anything that can be converted to a string, instead of insisting on strings.  Basically I'd like it possible to convert `'... {x} ...'.format(x)` to `'... {x} ...'.format(theme.thingy(x))` for any type of x.

Reply:
I converted it to Any.